packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6512,12 +6512,21 @@ export default class Meeting extends StatelessWebexPlugin {
    *
    * @private
    * @static
+   * @param {boolean} isAudioEnabled
+   * @param {boolean} isVideoEnabled
    * @returns {Promise<void>}
    */
-  private static async handleDeviceLogging(): Promise<void> {
+  private static async handleDeviceLogging(
+    isAudioEnabled = false,
+    isVideoEnabled = false
+  ): Promise<void> {
     try {
-      const devices = await getDevices();
+      const deviceKind =
+        (isVideoEnabled && Media.DeviceKind.VIDEO_INPUT) ||
```

Comment:
Does enabling video automatically ask for audio too?
What's the reason for undefined being present here?